### PR TITLE
Provider APIs updates

### PR DESCRIPTION
**Pull request by ConnectDIY**

### CODE REVIEW

File: th2_data_services/provider/v5/provider_api/grpc.py [hunk 1]
```diff
@@ -139,20 +197,50 @@ def search_messages(
...
+        if start_timestamp:
+            nanos = start_timestamp % 10 ** 9
+            seconds = start_timestamp // 10 ** 9
+            start_timestamp = Timestamp(seconds=seconds, nanos=nanos)
 
-        # TODO END
+        if end_timestamp:
+            nanos = end_timestamp % 10 ** 9
+            seconds = end_timestamp // 10 ** 9
+            end_timestamp = Timestamp(seconds=seconds, nanos=nanos)
```
Comment (ConnectDIY) [inline th2_data_services/provider/v5/provider_api/grpc.py:208]: @Svyat935 Let's remove duplicates

File: th2_data_services/provider/v5/provider_api/grpc.py [hunk 2]
```diff
@@ -109,23 +144,46 @@ def search_events(
...
             metadata_only=metadata_only,
             attached_messages=attached_messages,
+            filters=filters,
         )
         return self.__stub.searchEvents(event_search_request)
 
     def search_messages(
         self,
-        start_timestamp: datetime,
+        start_timestamp: int,
```
Comment (ConnectDIY) [inline th2_data_services/provider/v5/provider_api/grpc.py:153]: @Svyat935 
1. Let's add info that start/end timestamps are expected nanoseconds.
2. Add check, that these timestamps have length = 18 else raise an exception

**Comment by ConnectDIY** [comment]
@Svyat935 
Get_events and Get_messages look very similar.
Can we get rid of duplicates here too?

**Comment by Svyat935** [comment]
> @Svyat935 Get_events and Get_messages look very similar. Can we get rid of duplicates here too?
Цe can solve this with an entity class or some other way. But I don't know how.

**Review by Svyat935 — APPROVED** [review]

**Pull request merged**